model/Ssd1306.h

```diff
@@ -0,0 +1,167 @@
+#ifndef SSD1306_H
+#define SSD1306_H
+
+#include "Image.h"
+#include "MbedI2C.h"
+
+#include <string>
+
+
+namespace codal
+{
+   class Ssd1306 {
+   
+  
+   uint8_t Data[2];
+   uint8_t ScreenBuffer[1024];
+   int DeviceAddress = 0x78;
+   typedef unsigned char byte;
+
+     public:
+     codal::_mbed::I2C i2c;
+     Ssd1306(Pin& sda, Pin& scl);
+    
+    // Initialize the display
```

Comment:
Tighten up these declarations. Remove the comments and add proper doc comments that match the other classes